tck/api/src/main/java/io/astefanutti/metrics/cdi/se/CountedClassBean.java

```diff
@@ -15,9 +15,9 @@
  */
 package io.astefanutti.metrics.cdi.se;
 
-import org.eclipse.microprofile.metrics.annotation.Counted;
+import org.eclipse.microprofile.metrics.annotation.ParallelCounted;
 
-@Counted(name = "countedClass")
+@ParallelCounted(name = "countedClass")
```

Comment:
Better to keep Counted test cases and add new ones for ParallelCounted / HitCounted